crates/holochain_sqlite/src/db.rs

```diff
@@ -685,6 +690,7 @@ impl<'e> WriteManager<'e> for PConn {
         let mut txn = self
             .transaction_with_behavior(TransactionBehavior::Exclusive)
             .map_err(DatabaseError::from)?;
+        // TODO does not offload blocking work
```

Comment:
it *is* named `_sync` and this function is only available with feature `test_utils`, so maybe okay?

Reply:
Fair enough it is named `_sync` but it's used from an async context and in production code because `test_utils` is enabled by default for the `holochain_sqlite` crate

Reply:
For sure if it's used improperly out of testing contexts we should fix it.